spec/abstractops.html

```diff
@@ -196,62 +196,91 @@ <h1>ToShowOffsetOption ( _normalizedOptions_ )</h1>
+        It verifies that _increment_ evenly divides _dividend_, otherwise throwing a *RangeError*.
         _dividend_ must be divided into more than one part unless _inclusive_ is *true*.
       </dd>
     </dl>
     <emu-alg>
       1. If _inclusive_ is *true*, then
-        1. Let _maximum_ be 𝔽(_dividend_).
+        1. Let _maximum_ be _dividend_.
       1. Else if _dividend_ is more than 1, then
-        1. Let _maximum_ be 𝔽(_dividend_ - 1).
+        1. Let _maximum_ be _dividend_ - 1.
       1. Else,
-        1. Let _maximum_ be *1*<sub>𝔽</sub>.
+        1. Let _maximum_ be 1.
       1. If _increment_ &gt; _maximum_, throw a *RangeError* exception.
-      1. Set _increment_ to floor(ℝ(_increment_)).
       1. If _dividend_ modulo _increment_ is not zero, then
         1. Throw a *RangeError* exception.
-      1. Return _increment_.
     </emu-alg>
   </emu-clause>
 
-  <emu-clause id="sec-temporal-tosecondsstringprecision" aoid="ToSecondsStringPrecision">
-    <h1>ToSecondsStringPrecision ( _normalizedOptions_ )</h1>
-    <p>
-      The abstract operation ToSecondsStringPrecision combines the values of the options `smallestUnit` and `fractionalSecondDigits` to yield a precision for printing minutes and seconds to a string, and a rounding unit and increment.
-      The precision may be an integer 0 through 9 signifying a number of digits after the decimal point in the seconds, the string *"minute"* signifying not to print seconds at all, or the string *"auto"* signifying to drop trailing zeroes after the decimal point.
-    </p>
+  <emu-clause id="sec-temporal-tofractionalseconddigits" type="abstract operation">
+    <h1>
+      ToFractionalSecondDigits (
+        _normalizedOptions_: an Object,
+      ): either a normal completion containing either an integer or *"auto"*, or an abrupt completion
+    </h1>
+    <dl class="header">
+      <dt>description</dt>
+      <dd>It extracts the value of the property named *"fractionalSecondDigits"* from _normalizedOptions_ and makes sure it is a valid value for the option.</dd>
+    </dl>
+    <emu-alg>
+      1. Let _digitsValue_ be ? Get(_normalizedOptions_, *"fractionalSecondDigits"*).
+      1. If _digitsValue_ is *undefined*, set _digitsValue_ to *"auto"*.
+      1. If _digitsValue_ is not a Number, then
+        1. Set _digitsValue_ to ? ToString(_digitsValue_).
+        1. If _digitsValue_ is not *"auto"*, throw a *RangeError* exception.
+        1. Return *"auto"*.
+      1. If _digitsValue_ is *NaN*, *+&infin;*<sub>𝔽</sub>, or *-&infin;*<sub>𝔽</sub>, throw a *RangeError* exception.
+      1. Let _digitCount_ be truncate(ℝ(_digitsValue_)).
+      1. If _digitCount_ &lt; 0 or _digitCount_ &gt; 9, throw a *RangeError* exception.
+      1. Return _digitCount_.
+    </emu-alg>
+  </emu-clause>
+
+  <emu-clause id="sec-temporal-tosecondsstringprecision" type="abstract operation">
+    <h1>
+      ToSecondsStringPrecision (
```

Comment:
Suggested rename for clarity:
```suggestion
  <emu-clause id="sec-temporal-tosecondsstringprecisionrecord" type="abstract operation">
    <h1>
      ToSecondsStringPrecisionRecord (
```

Reply:
Done in a separate commit.